src/components/Cast/CastHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import { HeaderText, LinkText, SubHeaderText } from 'components/ui/Text'
+import { displayFrom } from 'helpers/visibilityClassnames'
+import classnames, {
+  display,
+  flexDirection,
+  gap,
+  width,
+} from 'classnames/tailwind'
+
+const castHeaderWrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  gap('gap-y-4')
+)
+
+const optionalTextWrapper = classnames(displayFrom('md'), display('md:!inline'))
+
+export default function () {
+  const farcasterEchoUrl =
+    'farcaster://profiles/0xAD81Fb97FEcaFA3A9830d4e2F424aC1776024DA8/posts'
```

Comment:
Why don't we move the address to `.env`?

Reply:
moved